### PR TITLE
SNI-4236: Added Legal officer template for orders

### PR DESCRIPTION
**Pull request by rajaram-nagarajan**
### Change description ###

new order template added

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SNI-4236

### CODE REVIEW

**Review by gmmagruder — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/model/OrderTemplate.java [hunk 1]
```diff
@@ -22,7 +22,10 @@ public enum OrderTemplate implements HasLabel {
     CIC10_STRIKE_OUT_WARNING("CIC10 - Strike Out Warning", "CIC10_Strike_Out_Warning"),
 
     @JsonProperty("CIC13_Pro_Forma_Summons")
-    CIC13_PRO_FORMA_SUMMONS("CIC13 - Pro Forma Summons", "CIC13_Pro_Forma_Summons");
+    CIC13_PRO_FORMA_SUMMONS("CIC13 - Pro Forma Summons", "CIC13_Pro_Forma_Summons"),
+
+    @JsonProperty("CIC14 – LO General Directions")
```
Comment (tomxelliott) [inline src/main/java/uk/gov/hmcts/sptribs/ciccase/model/OrderTemplate.java:27]: Do we want the Json property to follow the snake_case naming policy used elsewhere in this class?

**Review by tomxelliott — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/model/OrderTemplate.java (hunk 1 above)
Comment (rajaram-nagarajan) [reply src/main/java/uk/gov/hmcts/sptribs/ciccase/model/OrderTemplate.java:27]: yes, good catch

**Review by tomxelliott — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/model/UserRole.java [hunk 2]
```diff
@@ -12,7 +12,7 @@ public enum UserRole implements HasRole {
     // Common User roles
     //TODO: update : SUPER_USER, SOLICITOR before prod deploy
     SUPER_USER("caseworker-sptribs-superuser", "CRU"),
-    SYSTEMUPDATE("caseworker-st_cic-systemupdate", "CRUD"),
+    SYSTEMUPDATE("caseworker-sptribs-systemupdate", "CRUD"),
```
Comment (rajaram-nagarajan) [inline src/main/java/uk/gov/hmcts/sptribs/ciccase/model/UserRole.java:15]: **This change needs to be reverted before merging.**

Comment (rajaram-nagarajan) [reply src/main/java/uk/gov/hmcts/sptribs/ciccase/model/UserRole.java]: latest from master was merged.

**Review by tomxelliott — APPROVED** [review]

**Pull request merged**